src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/mat_mul.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "common_test_utils/test_enums.hpp"
+
+namespace ov {
+namespace test {
+struct ShapeRelatedParams {
+    std::pair<InferenceEngine::SizeVector, bool> input1, input2;
```

Comment:
Why we use such structure??? Looks like we can remove

Reply:
removed